src/lib/UtapiClient.js

```diff
@@ -339,7 +341,8 @@ export default class UtapiClient {
             }
             // number of objects counter
             const actionErr = results[0][0];
-            const actionCounter = results[0][1];
+            // If < 0, record numberOfObjects as though bucket were empty.
```

Comment:
I have to admit the logic of handling negative results escapes me, but I trust it's the right thing :)